vars/CentOS_8.yml

```diff
@@ -9,3 +9,5 @@ blivet_package_list:
   - vdo
   - kmod-kvdo
   - xfsprogs
+  - stratisd
+  - stratis-cli
```

Comment:
Does it make sense to separate packages, and only install once that are needed. E.g. for this use case, add a `blivet_stratis_packages` and install these packages only when user sets `stratis` variables?

Reply:
It's probably worth installing the dependencies a bit "smarter" -- we already have VDO here which also isn't that widely used.

Reply:
Note that blivet already has a way to return a list of package names that are required to perform the given actions - https://github.com/linux-system-roles/storage/blob/main/tasks/main-blivet.yml#L21 - can that be extended to include all of the packages we currently put in the various vars files, such as vdo, xfsprogs, etc.?